api.py

```diff
@@ -1,11 +1,97 @@
-from flask import Flask
```

Comment:
Appreciate this wasn't there before, but would it be worth adding type hints now that it's being rejigged a bit? Might be one for a future ticket though so no worries if not

Reply:
Worth adding type hints for this repo (and others) in their own tickets, so we can ensure they are progressed and reviewed properly. But def worth doing 👍